### ISSUE TITLE
Edit Mapping (Not Responding) after working for about 45 minutes

### ISSUE THREAD

**Issue by PhilSuiter** [ODM2/ODM2StreamingDataLoader#38]
After working for about 45 minutes creating and assigning results to the variables, the program would temporarily freeze and was "not responding".  After about one minute, I was able to cancel out and get to the main edit mapping page, but depending on where the program froze, I would have to recreate the result.  Sometimes I would have to click "Finish" and re-connect to the edit mapping in order to have the program work again.  Once the program froze the first time, it would reoccur about every 20 minutes.

**Comment by denvaar** [comment]
Has it stopped working after about 45 minutes on multiple separate instances?

**Comment by PhilSuiter** [comment]
Yes.  It has happened about 4 times after working about 45 minutes.  Then it happens more frequently after that. It's not a long term freeze where I can't do anything at all, but it will 'not respond' after clicking a button, then it will appear okay, but when I click 'finish' again, it will continue to 'not respond'.  I am usually able to click cancel and re-due the result.

**Comment by kwuz** [comment]
cannot reproduce -- do you have any more specific conditions I can try?
